### PR TITLE
Create translation_bg.json

### PR DESCRIPTION
**Pull request by hectorindi**
Added new language

### CODE REVIEW

**Comment by hectorindi** [comment]
Hi , 
We dont have any transletion for 'readyCompetitions' but we do have for 
Active Tournaments
Upcoming Tournaments
Finished Turnaments
Let me know if we should request a new translation

**Comment by hectorindi** [comment]
Hi,
I also noticed that file doesn't have translation for 'leaderboard' so i added 'label' field in the object. Let me know if you will be using or from where 'leaderboard' text is getting translation

File: src/i18n/translation_fa.json [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+        "minutesShortHand": "دقیقه",
+        "minutes": "دقیقه",
+        "seconds": "ثانیه"
+    },
+    "achievements": {
+        "label": "دستاوردها",
+        "more": "موارد بیشتر",
+        "complete": "100% کامل"
+    },
+    "tournaments": {
```
Comment (s-koval) [inline src/i18n/translation_fa.json:14]: "label": "Tournaments",

File: src/i18n/translation_id.json [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+        "minutesShortHand": "mnt",
+        "minutes": "menit",
+        "seconds": "detik"
+    },
+    "achievements": {
+        "label": "prestasi",
+        "more": "selengkapnya",
+        "complete": "selesaikan 100%"
+    },
+    "tournaments": {
```
Comment (s-koval) [inline src/i18n/translation_id.json:14]: "label": "Tournaments",

File: src/i18n/translation_my.json [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+        "minutesShortHand": "min",
+        "minutes": "minit",
+        "seconds": "saat"
+    },
+    "achievements": {
+        "label": "pencapaian",
+        "more": "lagi",
+        "complete": "lengkap 100%"
+    },
+    "tournaments": {
```
Comment (s-koval) [inline src/i18n/translation_my.json:14]: "label": "Tournaments",

**Review by s-koval — APPROVED** [review]

**Pull request merged**